lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -1097,6 +1101,14 @@
   .tab-content {
     @include padding($baseline $baseline 0 $baseline);
```

Comment:
NIT: no need to use the RTL ``include padding()`` mixin here (since all values are equal).